conjur/cli.py

```diff
@@ -173,7 +179,7 @@ def _parse_args(parser):
             parser.print_help()
             sys.exit(0)
 
-        if args.resource not in ['list']:
+        if args.resource not in ['list', 'whoami']:
```

Comment:
Maybe it would be good to add a comment that this is to allow no-arg commands from throwing an error?